kornia/geometry/boxes.py

```diff
@@ -55,6 +75,61 @@ def _boxes_to_polygons(
     return polygons
 
 
+def _boxes_to_quadrilaterals(
+    boxes: torch.Tensor, mode: str = "xyxy", validate_boxes: bool = True
+) -> torch.Tensor:
+    """Convert from boxes to quadrilaterals."""
```

Comment:
do we have somewhere a reference of the definition of quadrilaterals ?

Reply:
any four points will make a quadrilateral. It can be irregular shaped, while boxes have to be orthogonal